@navikt/aksel/src/codemod/transforms/v4.0.0/internal-css/internal-css.ts

```diff
@@ -0,0 +1,20 @@
+const cssConversions = {
+  ".navdsi-dropdown": ".navds-dropdown",
+  ".navdsi-header": ".navds-internalheader",
+  ".navdsi-timeline": ".navds-timeline",
+};
+/**
+ * @param {import('jscodeshift').FileInfo} file
+ * @param {import('jscodeshift').API} api
+ */
+export default function transformer(file) {
+  let src = file.source;
+
+  Object.entries(cssConversions).forEach(([key, value]) => {
+    // eslint-disable-next-line no-useless-concat
+    const rgx = new RegExp(`${key}`, "gm");
+    src = src.replace(rgx, `${value}`);
```

Comment:
1. Virker som eslint-disable er unødvendig?
2. Tror du eventuelt kan bruke `replaceAll()` for å slippe å bruke regex
3. Ser ut som det ikke er nødvendig med template literals